.travis.yml

```
@@ -9,6 +9,8 @@ env:
 
 before_install:
   - wget https://raw.githubusercontent.com/xtuple/xtuple-server-core/master/bootstrap.sh -qO- | sudo bash
+  - node --version
+  - n 0.11.13
```

This will hide a potential failure of bootstrap.sh to install and declare the correct version of node (0.11.13) on its own.